src/domain/users/wallets/wallets.repository.ts

```diff
@@ -0,0 +1,5 @@
+import { Injectable } from '@nestjs/common';
+import { IWalletsRepository } from '@/domain/users/wallets/wallets.repository.interface';
+
+@Injectable()
+export class WalletsRepository implements IWalletsRepository {}
```

Comment:
@iamacook I don't think this and its interface are currently needed. Should we remove them?

Reply:
Considering the wallet can be created via the entity manager directly, I think it's fine to remove it. I'll leave it up to you.